src/main/java/core/basesyntax/FileWork.java

```diff
@@ -8,7 +18,25 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+    private static final char FIRST_LETTER = 'w';
+
     public String[] readFromFile(String fileName) {
+        try (Stream<String> lines = Files.lines(Paths.get(fileName))) {
+            List<String> content = lines.collect(Collectors.toList());
+            Pattern p = Pattern.compile("(?i)\\b(" + FIRST_LETTER + "\\w*)\\b");
+            Matcher m = p.matcher(content.toString());
+            List<String> list = new ArrayList<>();
+
+            while (m.find()) {
+                list.add(m.group().toLowerCase());
+            }
+            String[] arr = new String[list.size()];
+            list.toArray(arr);
+            Arrays.sort(arr);
+            return arr;
+        } catch (Exception e) {
+            System.out.println(e.getMessage());
```

Comment:
here, you basically do nothing now. rethrow the Exception as a RuntimeException. see the checklist

Reply:
Done